charts/enterprise-metrics/exports/default/apps-v1.Deployment-enterprise-metrics-admin-api.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: enterprise-metrics-admin-api
+    app.kubernetes.io/managed-by: Helmraiser
```

Comment:
Is Helmraiser a real tool?

Reply:
Yeah! It's a cool feature of Tanka to work with Helm charts (https://github.com/grafana/tanka/commit/6d99591bf404393da3e0904cc8e04e7fda42a354)